docs/internals/howto-release-django.txt

```diff
@@ -51,18 +48,29 @@ There are a lot of details, so please read on.
 Prerequisites
 =============
 
-You'll need a few things hooked up to make this work:
+You'll need a few things before getting started:
 
-* A GPG key recorded as an acceptable releaser in the `Django releasers`__
-  document. (If this key is not your default signing key, you'll need to add
-  ``-u you@example.com`` to every GPG signing command below, where
+* A GPG key (If the key you want to use is not your default signing key, you'll
+  need to add ``-u you@example.com`` to every GPG signing command below, where
   ``you@example.com`` is the email address associated with the key you want to
   use.)
 
-* Access to Django's record on PyPI.
+* Access to Django's record on PyPI. Create a file with your credentials:
 
-* Access to the ``djangoproject.com`` server to upload files and trigger a
-  deploy.
+  .. snippet::
+    :filename: ~/.pypirc
+
+    [pypi]
+    username:YourUsername
+    password:YourPassword
+
+* An install of some required Python packages:
+
+  .. code-block:: bash
+
+      $ pip install wheel twine
```

Comment:
Seems to me this should be after the gpg key and before the PyPI access. That way the access permissions are separate from the things that you have control over yourself.